File: ExcelDivider/src/test/java/ru/tsystems/divider/components/impl/FileCleanerTest.java
package ru.tsystems.divider.components.impl;

import org.junit.Test;

import static org.junit.Assert.*;

public class FileCleanerTest {

    @Test
    public void tagsTest(){
        String str = "<a href=\"https://support.clicksoftware.com/csweb/downloads.nsf/cSearchView/A717CC68320D14724225746D00342E96?OpenDocument&amp;Highlight=PTV%2cDATA%2c17\">https://support.clicksoftware.com/csweb/downloads.nsf/cSearchView/A717CC68320D14724225746D00342E96?OpenDocument&amp;Highlight=PTV,DATA,17).</div>";
        FileCleaner.removeTags(str);
    }

}

File: ExcelDivider/src/test/kotlin/ru/tsystems/divider/utils/impl/FileCleanerTest.kt
package ru.tsystems.divider.utils.impl

import org.junit.Test
import kotlin.test.assertEquals

class FileCleanerTest {

    @Test
    fun tagsTest() { //todo: link from href should be replaced with link between tags
        val str =
            "<a href=\"https://support.clicksoftware.com/csweb/downloads.nsf/cSearchView/A717CC68320D14724225746D00342E96?OpenDocument&amp;Highlight=PTV%2cDATA%2c17\">https://support.clicksoftware.com/csweb/downloads.nsf/cSearchView/A717CC68320D14724225746D00342E96?OpenDocument&amp;Highlight=PTV,DATA,17).</div>"
        val expected =
            "https://support.clicksoftware.com/csweb/downloads.nsf/cSearchView/A717CC68320D14724225746D00342E96?OpenDocument&amp;Highlight=PTV,DATA,17).\n"
        val result = FileCleaner().removeTags(str)
        assertEquals(expected, result)
    }

}